plugins/woocommerce-blocks/tests/e2e/tests/related-products/related-products.block_theme.spec.ts

```diff
@@ -63,4 +63,54 @@ test.describe( `${ blockData.name } Block`, () => {
 			await editor.getBlockByName( blockData.slug )
 		).toBeVisible();
 	} );
+
+	test( "doesn't display additional Products (Beta) options", async ( {
+		page,
+		admin,
+		editor,
+		editorUtils,
```

Comment:
It seems that `editorUtils` does not exist anymore so it's causing an error in the E2E tests. You should use `editor` instead.

```suggestion
```